conversation/frontend/src/features/message-list/message-preview/components/RecipientListPreview.tsx

```diff
@@ -4,9 +4,13 @@ import { MessageMetadata } from '~/models';
 
 export interface RecipientListPreviewProps {
   message: MessageMetadata;
+  hasPrefix?: boolean;
 }
 
-export function RecipientListPreview({ message }: RecipientListPreviewProps) {
+export function RecipientListPreview({
+  message,
+  hasPrefix,
```

Comment:
```suggestion
  head,
```
ou `}, ...otherProps`